index.html

```diff
@@ -0,0 +1,498 @@
+        <h1 class="text-center">
+          The Best Software to Grow <br />
+          your Sales and Services
+        </h1>
+        <p class="header-container__paragraph">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor <br />
+          invidunt ut labore et dolore magna aliquyam erat.
+        </p>
+        <div class="header-container__inputs">
+          <input
+            type="text"
+            placeholder="Enter Your Email"
+            class="outline-none w-full px-4 font-roboto"
+          />
+          <button class="header-container__btn">Get Free trial</button>
+        </div>
+
+        <div class="header-container__chart">
+          <img
+            src="./assets/images/chart.png"
+            alt=""
+            class="header-container__graphique"
+          />
+        </div>
+      </div>
+    </header>
+    <!-- Header end -->
+    <!-- High-quality section start-->
+    <section class="high-quality">
+      <h4 class="high-quality__subtitle">High-quality</h4>
+      <h1 class="high-quality__title">
+        We have the Best Solution<br />
+        for your Business
+      </h1>
+
+      <div class="high-quality__card-container">
+        <div class="high-quality__card">
+          <img
+            src="./assets/images/feature-icon-1.svg"
+            alt="security"
+            class="high-quality__card-img"
+          />
+          <h3 class="high-quality__card-title">
+            High security to protect<br />
+            from piracy
+          </h3>
+          <p class="high-quality__card-description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing <br />
+            elitr, sed diam nonumy.
+          </p>
+        </div>
+        <div class="high-quality__card">
+          <img
+            src="./assets/images/feature-icon-2.svg"
+            alt="security"
+            class="high-quality__card-img"
+          />
+          <h3 class="high-quality__card-title">
+            Premium quality<br />
+            performance
+          </h3>
+          <p class="high-quality__card-description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing <br />
+            elitr, sed diam nonumy.
+          </p>
+        </div>
+        <div class="high-quality__card">
+          <img
+            src="./assets/images/feature-icon-3.svg"
+            alt="security"
+            class="high-quality__card-img"
+          />
+          <h3 class="high-quality__card-title">
+            Full time customer<br />
+            support - 24/7
+          </h3>
+          <p class="high-quality__card-description">
+            Lorem ipsum dolor sit amet, consetetur sadipscing <br />
+            elitr, sed diam nonumy.
+          </p>
+        </div>
+      </div>
+    </section>
+    <!-- High-quality section end -->
+    <!-- work-with-us section start -->
+    <section class="work-with-us">
+      <div class="work-with-us__left-content">
```

Comment:
_**left-content**_ is not a semantic name, because it just shows the location of the element on the page